diff -Nur -x '*.orig' -x '*.rej' e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/e-smith/sql/init/10fix_privilege_tables mezzanine_patched_e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/e-smith/sql/init/10fix_privilege_tables
--- e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/e-smith/sql/init/10fix_privilege_tables	2006-03-15 23:44:28.000000000 -0700
+++ mezzanine_patched_e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/e-smith/sql/init/10fix_privilege_tables	2007-04-21 15:21:42.000000000 -0600
@@ -1,6 +1,6 @@
 #!/bin/sh
 
-/usr/bin/mysql mysql < /usr/share/mysql/mysql_fix_privilege_tables.sql
+/usr/bin/mysql -f mysql < /usr/share/mysql/mysql_fix_privilege_tables.sql
 P=$(cat /var/run/mysqld/mysqld.pid)
 /bin/rm /var/run/mysqld/mysqld.pid
 kill -TERM $P
