--- e-smith-mysql-2.0.0/root/etc/e-smith/templates/etc/my.cnf/010mysqld-log-error.FixErrorLogDetectionAlgorithm	2003-12-18 23:10:32.000000000 +0100
+++ e-smith-mysql-2.0.0/root/etc/e-smith/templates/etc/my.cnf/010mysqld-log-error	2008-12-02 16:12:20.000000000 +0100
@@ -5,7 +5,7 @@
     if (-f "/usr/sbin/mysqld") {
 	$mysqld = "/usr/sbin/mysqld";
     }
-    open (MYSQLD, "$mysqld --help |")
+    open (MYSQLD, "$mysqld --help --verbose |")
 	or return "# WARN: can't exec mysqld; skipping log-error setup";
     $LogError = grep {/log-error/} <MYSQLD>;
     close MYSQLD;
