--- e-smith-mysql-1.13.0/root/etc/rc.d/init.d/mysql.init.logging	2006-03-16 01:44:28.000000000 -0500
+++ e-smith-mysql-1.13.0/root/etc/rc.d/init.d/mysql.init	2008-10-06 10:55:19.000000000 -0400
@@ -45,11 +45,17 @@
 	    F=$(basename $link | sed s/S[0-9][0-9]//)
 	    case $F in
 		*.sql)
-		action "Loading $F into mysql" /usr/bin/mysql < $link && /bin/rm $link
-		;;
+                action "Loading $F into mysql" perl -e '
+                     open (STDERR, "|/usr/bin/logger -p local1.info -t mysql.init");
+                     open (STDOUT, ">&STDERR");
+                     exec "/usr/bin/mysql";' < $link && /bin/rm $link
+                ;;
 		*)
-		action "Loading $F into mysql" $link && /bin/rm $link
-		;;
+                action "Loading $F into mysql" perl -e '
+                     open (STDERR, "|/usr/bin/logger -p local1.info -t mysql.init");
+                     open (STDOUT, ">&STDERR");
+                     exec shift; ' $link && /bin/rm $link
+                ;;
 	    esac
 	    # Record any failure for the final return value.
 	    if [ $? -ne 0 ]; then
