diff -Nur -x '*.orig' -x '*.rej' e-smith-mysql-1.11.1/root/etc/e-smith/templates/etc/e-smith/sql/init/00_restore_dumped_dbs mezzanine_patched_e-smith-mysql-1.11.1/root/etc/e-smith/templates/etc/e-smith/sql/init/00_restore_dumped_dbs
--- e-smith-mysql-1.11.1/root/etc/e-smith/templates/etc/e-smith/sql/init/00_restore_dumped_dbs	1969-12-31 19:00:00.000000000 -0500
+++ mezzanine_patched_e-smith-mysql-1.11.1/root/etc/e-smith/templates/etc/e-smith/sql/init/00_restore_dumped_dbs	2006-01-01 21:01:01.000000000 -0500
@@ -0,0 +1,20 @@
+#! /bin/sh
+
+( cat /home/e-smith/db/mysql/mysql.dump ;
+  echo "use mysql; ALTER TABLE user MODIFY Password char(41) NOT NULL default '';" ;
+  cat /service/mysqld/set.password ) | mysql
+P=$(cat /var/run/mysqld/mysqld.pid)
+/bin/rm /var/run/mysqld/mysqld.pid
+kill -TERM $P
+for i in $(seq 1 20);
+do
+  if [ -f /var/run/mysqld/mysqld.pid ]
+  then
+      /bin/rm /home/e-smith/db/mysql/mysql.dump
+      exit 0
+  fi
+  echo waiting for mysqld to restart
+  sleep 1
+done
+echo mysqld failed to restart
+exit 1
diff -Nur -x '*.orig' -x '*.rej' e-smith-mysql-1.11.1/root/etc/e-smith/templates/etc/e-smith/sql/init/10fix_privilege_tables mezzanine_patched_e-smith-mysql-1.11.1/root/etc/e-smith/templates/etc/e-smith/sql/init/10fix_privilege_tables
--- e-smith-mysql-1.11.1/root/etc/e-smith/templates/etc/e-smith/sql/init/10fix_privilege_tables	2006-01-01 21:03:25.000000000 -0500
+++ mezzanine_patched_e-smith-mysql-1.11.1/root/etc/e-smith/templates/etc/e-smith/sql/init/10fix_privilege_tables	2006-01-01 20:59:55.000000000 -0500
@@ -1,5 +1,17 @@
 #!/bin/sh
 
-MYSQL_PW=`sed q /etc/openldap/ldap.pw`
-/usr/bin/mysql_fix_privilege_tables --password=$MYSQL_PW
-/usr/bin/mysqladmin shutdown
+/usr/bin/mysql mysql < /usr/share/mysql/mysql_fix_privilege_tables.sql
+P=$(cat /var/run/mysqld/mysqld.pid)
+/bin/rm /var/run/mysqld/mysqld.pid
+kill -TERM $P
+for i in $(seq 1 20);
+do
+  if [ -f /var/run/mysqld/mysqld.pid ]
+  then
+      exit 0
+  fi
+  echo waiting for mysqld to restart
+  sleep 1
+done
+echo mysqld failed to restart
+exit 1
diff -Nur -x '*.orig' -x '*.rej' e-smith-mysql-1.11.1/root/etc/e-smith/templates.metadata/etc/e-smith/sql/init/00_restore_dumped_dbs mezzanine_patched_e-smith-mysql-1.11.1/root/etc/e-smith/templates.metadata/etc/e-smith/sql/init/00_restore_dumped_dbs
--- e-smith-mysql-1.11.1/root/etc/e-smith/templates.metadata/etc/e-smith/sql/init/00_restore_dumped_dbs	1969-12-31 19:00:00.000000000 -0500
+++ mezzanine_patched_e-smith-mysql-1.11.1/root/etc/e-smith/templates.metadata/etc/e-smith/sql/init/00_restore_dumped_dbs	2006-01-01 21:01:26.000000000 -0500
@@ -0,0 +1 @@
+PERMS=0540
diff -Nur -x '*.orig' -x '*.rej' e-smith-mysql-1.11.1/root/var/service/mysqld/run mezzanine_patched_e-smith-mysql-1.11.1/root/var/service/mysqld/run
--- e-smith-mysql-1.11.1/root/var/service/mysqld/run	2006-01-01 21:03:25.000000000 -0500
+++ mezzanine_patched_e-smith-mysql-1.11.1/root/var/service/mysqld/run	2006-01-01 21:02:30.000000000 -0500
@@ -7,9 +7,7 @@
     /usr/libexec/mysqld --bootstrap --user=mysql --skip-grant-tables < ./set.password
     if [ -f /home/e-smith/db/mysql/mysql.dump ]
     then
-	mv /home/e-smith/db/mysql/mysql.dump /etc/e-smith/sql/init/00_restore_dumped_dbs.sql
-	echo "use mysql; ALTER TABLE user MODIFY Password char(41) NOT NULL default '';" >> /etc/e-smith/sql/init/00_restore_dumped_dbs.sql
-	cat ./set.password >> /etc/e-smith/sql/init/00_restore_dumped_dbs.sql
+	/sbin/e-smith/expand-template /etc/e-smith/sql/init/00_restore_dumped_dbs
     fi
 fi
 
