diff -Nur -x '*.orig' -x '*.rej' e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/011user mezzanine_patched_e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/011user
--- e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/011user	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/011user	2007-01-11 07:49:45.000000000 -0700
@@ -0,0 +1,8 @@
+{   
+    # The 'user' option must be in msyqld_safe/safe_mysqld section to avoid a bogus
+    # mysqld warning in the error log.  See http://bugs.mysql.com/2163
+    # NOTE: This problem is fixed in mysql >= 4.0.18, so once we get there we
+    # should move the 'user' option back to the [mysqld] section, and remove it
+    # from any mysqld invocations in our scripts.
+    $OUT = "user=mysql" if $LogError;
+}
diff -Nur -x '*.orig' -x '*.rej' e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/041user mezzanine_patched_e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/041user
--- e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/041user	2003-12-24 08:45:42.000000000 -0700
+++ mezzanine_patched_e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/041user	2007-01-11 07:50:08.000000000 -0700
@@ -1,6 +1,8 @@
-# The 'user' option must be in msyqld_safe/safe_mysqld section to avoid a bogus
-# mysqld warning in the error log.  See http://bugs.mysql.com/2163
-# NOTE: This problem is fixed in mysql >= 4.0.18, so once we get there we
-# should move the 'user' option back to the [mysqld] section, and remove it
-# from any mysqld invocations in our scripts.
-user=mysql
+{   
+    # The 'user' option must be in msyqld_safe/safe_mysqld section to avoid a bogus
+    # mysqld warning in the error log.  See http://bugs.mysql.com/2163
+    # NOTE: This problem is fixed in mysql >= 4.0.18, so once we get there we
+    # should move the 'user' option back to the [mysqld] section, and remove it
+    # from any mysqld invocations in our scripts.
+    $OUT = "user=mysql" unless $LogError;
+}
