diff -Nur -x '*.orig' -x '*.rej' e-smith-mysql-1.11.1/createlinks mezzanine_patched_e-smith-mysql-1.11.1/createlinks
--- e-smith-mysql-1.11.1/createlinks	2005-11-18 18:25:38.577280875 +1100
+++ mezzanine_patched_e-smith-mysql-1.11.1/createlinks	2005-11-18 18:25:18.499991833 +1100
@@ -45,3 +45,6 @@
 $event = "timezone-update";
 
 safe_symlink("restart", "root/etc/e-smith/events/$event/services2adjust/mysqld");
+
+safe_symlink("/var/service/mysqld/fix_privilege_tables",
+	"root/etc/e-smith/sql/init/S99fix_privilege_tables");
diff -Nur -x '*.orig' -x '*.rej' e-smith-mysql-1.11.1/root/var/service/mysqld/fix_privilege_tables mezzanine_patched_e-smith-mysql-1.11.1/root/var/service/mysqld/fix_privilege_tables
--- e-smith-mysql-1.11.1/root/var/service/mysqld/fix_privilege_tables	1970-01-01 10:00:00.000000000 +1000
+++ mezzanine_patched_e-smith-mysql-1.11.1/root/var/service/mysqld/fix_privilege_tables	2005-11-18 18:22:57.316116583 +1100
@@ -0,0 +1,4 @@
+#!/bin/sh
+
+MYSQL_PW=`sed q /etc/openldap/ldap.pw`
+/usr/bin/mysql_fix_privilege_tables --password=$MYSQL_PW
