diff -Nur -x '*.orig' -x '*.rej' e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/009innodb mezzanine_patched_e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/009innodb
--- e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/009innodb	2002-09-10 15:34:51.000000000 -0400
+++ mezzanine_patched_e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/009innodb	2006-04-30 13:23:05.000000000 -0400
@@ -1 +1,19 @@
-skip-innodb
+{
+    my $innodb = $mysqld{InnoDB} || 'disabled';
+
+    return "skip-innodb" unless $innodb eq 'enabled';
+
+    # innodb is enabled. We'll use the MySQL-AB-recommended innodb
+    # defaults for small to medium sites, from the my-medium.cnf file
+    # distributed with MySQL
+    $OUT .= "innodb_data_home_dir = /var/lib/mysql/\n";
+    $OUT .= "innodb_data_file_path = ibdata1:10M:autoextend\n";
+    $OUT .= "innodb_log_group_home_dir = /var/lib/mysql/\n";
+    $OUT .= "innodb_log_arch_dir = /var/lib/mysql/\n";
+    $OUT .= "innodb_buffer_pool_size = 16M\n";
+    $OUT .= "innodb_additional_mem_pool_size = 2M\n";
+    $OUT .= "innodb_log_file_size = 5M\n";
+    $OUT .= "innodb_log_buffer_size = 8M\n";
+    $OUT .= "innodb_flush_log_at_trx_commit = 1\n";
+    $OUT .= "innodb_lock_wait_timeout = 50\n";
+}
