diff -Nur -x '*.orig' -x '*.rej' e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/011max_allowed_packet mezzanine_patched_e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/011max_allowed_packet
--- e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/011max_allowed_packet	1969-12-31 17:00:00.000000000 -0700
+++ mezzanine_patched_e-smith-mysql-1.12.0/root/etc/e-smith/templates/etc/my.cnf/011max_allowed_packet	2007-04-09 02:56:41.000000000 -0600
@@ -0,0 +1 @@
+max_allowed_packet=16M
